### ISSUE TITLE
Brackets scrolls up a few lines when saving a file while the PHP Code Quality Tools panel is open

### ISSUE THREAD

**Issue by hjoelr** [mikaeljorhult/brackets-php-code-quality-tools#29]
I noticed a peculiar issue in Brackets where upon file save, my code view would scroll up a few lines from where I was working. I noticed this especially when I was working at the bottom of a file. I can't remember if it does it if I'm somewhere above the end of the file. I found that when I closed the PHP Code Quality Tools panel, this strange scrolling behavior ceased. That led me to believe that it had something to do with this extension. Closing the panel gave me another problem where [I can't figure out how to reopen the panel](https://github.com/mikaeljorhult/brackets-php-code-quality-tools/issues/28).

**Comment by mikaeljorhult** [comment]
That sounds like a bug in Brackets itself as this extension only doesn't modify the interface at all just providing Brackets with a list of errors. I'll see if I can recreate it and forward the bug.

Which version of Brackets are you using and on what OS?

**Comment by hjoelr** [comment]
@mikaeljorhult This would appear to be a Brackets issue since your extension uses Bracket's built-in panel. I'm on Brackets Release 1.6 build 1.6.0-16680 (release 3df0ac6fa) on Windows 10. Another factor in this is that I have the [Brackets Git extension](https://github.com/zaggino/brackets-git) installed with the panel open underneath the Lint panel as I show here:

![image](https://cloud.githubusercontent.com/assets/221971/13429232/3faecade-df8d-11e5-9f9c-2d2c1ebe38c8.png)

Maybe a combination of both those panels is where the issue happens. I'll do a little more experimenting and see if I can be a little more precise in what I'm saying (now that I know how to open the Lint panel).

**Comment by hjoelr** [comment]
@mikaeljorhult I spent some time poking around with this and found that it did not relate to having both the GIT panel and the Brackets Lint panel open. I was able to replicate the issue with just the Lint panel open. To more clearly demonstrate the issue, I put together a [short video](https://youtu.be/D-7kHHPoEkI). I'm not certain at this point if it's a Brackets issue or an issue somewhere in this extension. The peculiar thing is what happens when there are duplicate functions in the document. Let me know if there's anything else I can do to help stamp this bug out. Thanks!
